docs/site/Loopback-component-authentication.md

```diff
@@ -0,0 +1,600 @@
+---
+lang: en
+title: 'Authentication'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Loopback-component-authentication.html
+---
+
+## Overview
+
+Security is of paramount importance when developing a web or mobile application
+and usually consists of two distinct pieces :
+
+- Authentication
+- Authorization
+
+`Authentication` is a process of verifying someone's identity before a protected
+endpoint is accessed.
+
+`Authorization` is a process of verifying the actions that an authenticated user
+is allowed to perform on a protected endpoint.
+
+This document describes the details of the LoopBack 4 `Authentication`
+component.
+
+{% include note.html content="
+For a description of an `Authorization` process, please see <a href='https://loopback.io/doc/en/lb4/Loopback-component-authorization.html'>Authorization</a>.
+" %}
+
+## Authentication Component
+
+To utilize `authentication` in an application `application.ts`, you must load
+the authentication component named `AuthenticationComponent`.
+
+```ts
+export class MyApplication extends BootMixin(
+  ServiceMixin(RepositoryMixin(RestApplication)),
+) {
+  constructor(options?: ApplicationConfig) {
+    super(options);
+
+    ...
```

Comment:
@bajtos suggested, at some point, that we should make the `ts` code snippet a valid one. so, my suggested change would be:
```
// ... 
```

instead of simply 
```...```